akka-stream-tests/src/test/scala/akka/stream/scaladsl/GraphZipLatestSpec.scala

```diff
@@ -49,6 +49,26 @@ class GraphZipLatestSpec
       probe.expectNext((true, 1))
     }
 
+    "emits as soon as one source is available" in {
+      val (probe, bools, ints) = testGraph[Boolean, Int]
+
+      Given("request for 3 elements")
+      probe.request(3)
+
+      And("a first element pushed on either source")
+      bools.sendNext(true)
+      ints.sendNext(1)
+
+      And("then 2 elements pushed only on one source")
+      ints.sendNext(2)
+      ints.sendNext(3)
+
+      Then("3 elements are emitted")
+      probe.expectNext((true, 1))
+      probe.expectNext((true, 2))
+      probe.expectNext((true, 3))
+    }
```

Comment:
So to cover the instance equality this would be `true` `true` on source one and 1 1 1 on source 2. And you would see three tuples emitted, because there was three separate elements coming in.

Reply:
ok I will change this test to cover this